src/mixed-reality/azext_mixed_reality/custom.py

```diff
@@ -7,171 +7,77 @@
-
-
-def remote_rendering_account_update(cmd,
-                                    resource_group_name,
-                                    account_name,
-                                    location=None,
-                                    tags=None,
-                                    sku=None,
-                                    kind=None,
-                                    storage_account_name=None):
-    remote_rendering_account = {}
-    remote_rendering_account['tags'] = tags
-    remote_rendering_account['location'] = location
-    remote_rendering_account['identity'] = json.loads("{\"type\": \"SystemAssigned\"}")
-    remote_rendering_account['sku'] = sku
-    remote_rendering_account['kind'] = kind
-    remote_rendering_account['storage_account_name'] = storage_account_name
-    client = cf_remote_rendering_account(cmd.cli_ctx)
-    return client.update(resource_group_name=resource_group_name,
-                         account_name=account_name,
-                         remote_rendering_account=remote_rendering_account)
-
-
-def remote_rendering_account_delete(cmd,
-                                    resource_group_name,
-                                    account_name):
-    client = cf_remote_rendering_account(cmd.cli_ctx)
-    return client.delete(resource_group_name=resource_group_name,
-                         account_name=account_name)
-
-
-def remote_rendering_account_list_key(cmd,
-                                      resource_group_name,
-                                      account_name):
-    client = cf_remote_rendering_account(cmd.cli_ctx)
-    return client.list_keys(resource_group_name=resource_group_name,
-                            account_name=account_name)
-
-
-def remote_rendering_account_regenerate_key(cmd,
-                                            resource_group_name,
-                                            account_name,
-                                            key=None):
-    regenerate = {}
-    regenerate['serial'] = ['primary', 'secondary'].index(key) + 1
-    client = cf_remote_rendering_account(cmd.cli_ctx)
-    return client.regenerate_keys(resource_group_name=resource_group_name,
-                                  account_name=account_name,
-                                  regenerate=regenerate)
+from azure.cli.core.aaz import has_value
+from .aaz.latest.spatial_anchors_account import Create as _SpatialAnchorsCreate
+from .aaz.latest.spatial_anchors_account.key import Renew as _SpatialAnchorsKeyRenew
+from .aaz.latest.remote_rendering_account import Create as _RemoteRenderingCreate
+from .aaz.latest.remote_rendering_account.key import Renew as _RemoteRenderingKeyRenew
+from knack.log import get_logger
+
+
+logger = get_logger(__name__)
+
+
+class SpatialAnchorsCreate(_SpatialAnchorsCreate):
+    def pre_operations(self):
+        args = self.ctx.args
+        if has_value(args.kind):
+            args.sku = args.kind
+            del args.kind
```

Comment:
Why need to delete the kind?

Reply:
It would raise an exception if sent with request to service.